File: Java/프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.java
class Solution {
    public String[] solution(String[] strArr) {
        String[] answer = new String[strArr.length];
        for(int i=0 ; i<answer.length; i++){
            answer[i]="";
        }
        for(int i=0; i<answer.length;i++){
            char ch[] = strArr[i].toCharArray();
            for(int j=0 ; j<ch.length ;j++){
                if(i%2==0) {
                    answer[i]+=Character.toLowerCase(ch[j]);
                }
                else {
                    answer[i]+=Character.toUpperCase(ch[j]);
                }
            }
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.js
function solution(strArr) {
    var answer = new Array(strArr.length);
    for(var i=0; i<answer.length;i++){
        answer[i]='';
    }
    for(var i=0; i<strArr.length; i++){
        if(i%2==0){
            for(var j=0; j<strArr[i].length;j++){
                answer[i]+=(strArr[i].charAt(j)).toLowerCase();
            }
        }
        else {
            for(var k=0; k<strArr[i].length;k++){
                answer[i]+=(strArr[i].charAt(k)).toUpperCase();
            } 
        }
    }
    return answer;
}